stripe/util.py

```diff
@@ -366,9 +366,13 @@ class class_method_variant(object):
     def __init__(self, class_method_name):
         self.class_method_name = class_method_name
 
-    def __call__(self, method):
+    T = TypeVar("T")
+
+    method: Any
```

Comment:
Callable?

Reply:
That causes problems in the implementation of `__call__` and I timed out trying to figure out how to resolve.